_includes/checkout-v3-resource-models.md

```diff
@@ -571,18 +582,114 @@ Content-Type: application/json
       "nonPaymentToken" : "12345678-1234-1234-1234-1234567890AB",
       "externalNonPaymentToken" : "1234567890",
       "cardBrand": "Visa",
-6     "cardType": "Credit",
-7     "maskedPan": "492500******0004",
-8     "expiryDate": "12/2022",
-9     "issuerAuthorizationApprovalCode": "L00302",
-10    "acquirerTransactionType": "STANDARD",
-11    "acquirerStan": "302",
-12    "acquirerTerminalId": "70101301389",
-13    "acquirerTransactionTime": "2022-06-15T14:12:55.029Z",
-14    "transactionInitiator": "CARDHOLDER"
+      "cardType": "Credit",
+      "maskedPan": "492500******0004",
+      "expiryDate": "12/2022",
+      "issuerAuthorizationApprovalCode": "L00302",
+      "acquirerTransactionType": "STANDARD",
+      "acquirerStan": "302",
+      "acquirerTerminalId": "70101301389",
+      "acquirerTransactionTime": "2022-06-15T14:12:55.029Z",
+      "transactionInitiator": "CARDHOLDER"
     }
   }
-}
+```
+
+```http
+   "paid": {
+    "id": "/psp/paymentorders/1f8d409e-8d8c-4ba1-a3ab-08da8caf7918/paid",
+    "instrument": "ApplePay",
+    "number": 80100001190,
+    "payeeReference": "1662360210",
+    "amount": 1500,
+    "details": {
+        "cardBrand": "Visa",
+        "cardType": "Credit",
+        "expiryDate": "12/0023",
+        "issuerAuthorizationApprovalCode": "L00392",
+        "acquirerTransactionType": "WALLET",
+        "acquirerStan": "392",
+        "acquirerTerminalId": "80100001190",
+        "acquirerTransactionTime": "2022-09-05T06:45:40.322Z",
+        "transactionInitiator": "CARDHOLDER"
+    }
+  }
+```
```

Comment:
I think we can afford to expand the example into valid JSON with an appropriate header for each example. Something like this.

````suggestion
{:.code-view-header}
**Apple Pay**

```http
HTTP/1.1 200 OK
Content-Type: application/json

{
  "paymentOrder": "/psp/paymentorders/5adc265f-f87f-4313-577e-08d3dca1a26c",
  "paid": {
    "id": "/psp/paymentorders/1f8d409e-8d8c-4ba1-a3ab-08da8caf7918/paid",
    "instrument": "ApplePay",
    "number": 80100001190,
    "payeeReference": "1662360210",
    "amount": 1500,
    "details": {
        "cardBrand": "Visa",
        "cardType": "Credit",
        "expiryDate": "12/0023",
        "issuerAuthorizationApprovalCode": "L00392",
        "acquirerTransactionType": "WALLET",
        "acquirerStan": "392",
        "acquirerTerminalId": "80100001190",
        "acquirerTransactionTime": "2022-09-05T06:45:40.322Z",
        "transactionInitiator": "CARDHOLDER"
    }
  }
}
```
````